tests/test_move.py

```diff
@@ -60,3 +88,88 @@ def test_move_dir(self):
         if self.preserve_time:
             dst_file2_info = dst_fs.getinfo("bar/baz.txt", namespaces)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
+
+
+class TestMove(unittest.TestCase):
+    def test_move_file_tempfs(self):
+        with open_fs("temp://") as src, open_fs("temp://") as dst:
+            src_dir = src.makedir("Some subfolder")
+            src_dir.writetext("file.txt", "Content")
+            dst_dir = dst.makedir("dest dir")
+            fs.move.move_file(src_dir, "file.txt", dst_dir, "target.txt")
+            self.assertEqual(dst.readtext("dest dir/target.txt"), "Content")
+            self.assertFalse(src.exists("Some subfolder/file.txt"))
```

Comment:
Only a minor quibble, but some of the tests check the "new state" of the src-fs first, and some check the state of the dest-fs first. The tests might be easier to read if you always checked the FSes in a consistent order?